apps/src/musicMenu/MusicMenu.jsx

```diff
@@ -80,8 +80,7 @@ const optionsList = [
   {
     name: 'key',
     type: 'string',
-    description:
-      'Set a specific key, as a number from 0-11. Starting at C = 0 (C# = 1, D = 2, D# = 3, ...)',
+    description: 'Set a specific key.',
```

Comment:
Worth capturing the format still?

Reply:
Ah yeah, I removed this since I switched this from a number to the key string ("C#", etc) but I can specify